### DISCUSSION TITLE
Database configuration fails "1067 Invalid default value for 'date_tz'"

### DISCUSSION THREAD

**Discussion by Linuxmaker** [kimai/kimai#2943] [On-Premise: Installation & Updates] [answered]
Hi there,
I get the following MySQL error while installing Kamai 1.16.1: 

```
Kimai installation running ...
==============================

Created database `kimai2` for connection named default
[notice] Migrating up to DoctrineMigrations\Version20211008092010
[error] Migration DoctrineMigrations\Version20210802160837 failed during Execution. Error: "An exception occurred while executing 'ALTER TABLE kimai2_timesheet CHANGE date_tz date_tz DATE DEFAULT 'NULL' NOT NULL':

SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'date_tz'"
                                                                                                                        
 [ERROR] Failed to set migration status: An exception occurred while executing 'ALTER TABLE kimai2_timesheet CHANGE     
         date_tz date_tz DATE DEFAULT 'NULL' NOT NULL':                                                                 
                                                                                                                        
         SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'date_tz'
```
This is what happens when I run the recommended command: 
`# bin/console kimai:install -n`
I followed the installation instructions and I use 
`mysql  Ver 15.1 Distrib 10.5.11-MariaDB, for debian-linux-gnu (x86_64)`
on the system 
`Linux srv3 5.10.0-8-amd64 #1 SMP Debian 5.10.46-4 (2021-08-03) x86_64 GNU/Linux`
Maybe you can tell me what else is needed for the installation.

Best Regards 

Andreas

**Comment by kevinpapst** [comment, marked as answer]
https://github.com/kevinpapst/kimai2/issues?q=is%3Aissue+date_tz+is%3Aclosed

**Reply by Linuxmaker** [reply]
Thank you, for the sake of correctness, here is the specific solution for everyone else: 
`
&serverVersion=mariadb-10.5.11`
in file .env

**Reply by kevinpapst** [reply]
Too many people with the same problem always means trouble for me 😁 
So I am investigating, why the heck the framework does not drop the default value on that column, without the proper server version in the connection string (which is a good idea anyways, but wasn't necessary in the past).

**Reply by Linuxmaker** [reply]
You can avoid the stress if this problem of the database is explicitly described in the HowTo. This is not pointed out clearly enough in the comments in the configuration file. That is easily overlooked, and so do I.
